iOS/_Stream Temp/ChatMessage+Extensions.swift

```diff
@@ -158,9 +148,17 @@ extension Message: Messageable {
         return MessageController.controller(cid, messageId: messageId).message!
     }
     
-    func setToConsumed() async throws {
+    func addEmotions(emtions: [Emotion]) async {
+        guard let controller = ChatClient.shared.messageController(for: self) else { return }
+        
+        await emotions.asyncForEach { emotion in
```

Comment:
I don't have a throwing async for each func yet, and we don't really care if it throws.